diffrax/_solver/langevin_srk.py

```diff
@@ -0,0 +1,325 @@
+import abc
+from typing import Any, Callable, Generic, TypeVar
+
+import equinox as eqx
+import equinox.internal as eqxi
+import jax.lax as lax
+import jax.numpy as jnp
+import jax.tree_util as jtu
+from equinox import AbstractVar
+from jax import vmap
+from jax._src.tree_util import PyTreeDef
+from jaxtyping import Array, ArrayLike, PyTree
+
+from .._custom_types import (
+    AbstractBrownianIncrement,
+    BoolScalarLike,
+    DenseInfo,
+    RealScalarLike,
+)
+from .._local_interpolation import LocalLinearInterpolation
+from .._solution import RESULTS
+from .._term import (
+    _LangevinDiffusionTerm,
+    _LangevinDriftTerm,
+    _LangevinTuple,
+    _LangevinX,
```

Comment:
If we're going to import these then we should make them public.

Reply:
Makes sense. I assume you mean just removing underscores, not adding them to `__init__.py`?